packages/graph/hash_graph/lib/graph/src/ontology/entity_type.rs

```diff
@@ -38,6 +38,33 @@ impl QueryRecord for EntityType {
     type Path<'q> = EntityTypeQueryPath;
 }
 
+impl OntologyPath for EntityTypeQueryPath {
+    /// Returns the path identifying the base URI.
```

Comment:
These docstrings are only on the `EntityTypeQueryPath` impl here. Is this intentional for doc generation, or could we move it onto the trait and let it inherit?

Reply:
Good spot, this was by accident: ee840bf9fb5534d2bad9be32d174f4719300c580